tests/test_examples.py

```diff
@@ -3,117 +3,310 @@
+
+
+def test_csci_1():
+    exampletest_single("CSCI-4968-MBE/challenges/crackme0x01")
+
+
+def test_csci_2():
+    exampletest_single("CSCI-4968-MBE/challenges/crackme0x02")
+
+
+def test_csci_3():
+    exampletest_single("CSCI-4968-MBE/challenges/crackme0x03")
+
+
+def test_csci_4():
+    exampletest_single("CSCI-4968-MBE/challenges/crackme0x04")
+
+
+def test_csci_5():
+    exampletest_single("CSCI-4968-MBE/challenges/crackme0x05")
+
+
+def test_insomnihack_aeg():
+    exampletest_single("insomnihack_aeg")
+
+
+def test_android_license():
+    exampletest_single("android_arm_license_validation")
+
+
+def test_sym_write():
+    exampletest_single("sym-write")
+
+
+def test_angry_reverser():
+    exampletest_single("hackcon2016_angry-reverser")
+
+
+def test_sharif7():
+    exampletest_single("sharif7_rev50")
+
+
+def test_angrybird():
+    exampletest_single("codegate_2017-angrybird")
+
+
+@slow_test
+def test_mbrainfuzz():
+    exampletest_single("secuinside2016mbrainfuzz")  # 1m46s
+
+
+def test_unmapped_analysis():
+    exampletest_single("unmapped_analysis")
+
+
+@slow_test
+def test_zwiebel():
+    exampletest_single("tumctf2016_zwiebel")  # ~45m
+
+
+def test_csgames2018():
+    exampletest_single("csgames2018")
+
+
+def test_java_crackme1():
+    exampletest_single("java_crackme1")
+
+
+def test_java_simple3():
+    exampletest_single("java_simple3")
+
+
+def test_java_simple4():
+    exampletest_single("java_simple4")
+
+
+def test_ictf2017_javaisnotfun():
+    exampletest_single("ictf2017_javaisnotfun")
+
+
+def test_java_androidnative1():
+    exampletest_single("java_androidnative1")
+
+
+@slow_test
+def test_defcon2019quals_veryandroidoso():
+    exampletest_single("defcon2019quals_veryandroidoso")
+
+
 ## END EXAMPLE TESTS
 
+
 def test_example_inclusion():
-    to_test = subprocess.check_output(['/bin/bash', '-c', 'for c in $(find -name solve.py | cut -c 3-); do echo ${c%/solve.py}; done'], cwd=_path('examples')).decode()
-    if __file__.endswith('.pyc'):
+    to_test = subprocess.check_output(
+        [
+            "/bin/bash",
+            "-c",
+            "for c in $(find -name solve.py | cut -c 3-); do echo ${c%/solve.py}; done",
+        ],
+        cwd=_path("examples"),
+    ).decode()
+    if __file__.endswith(".pyc"):
         sourcefile = __file__[:-1]
     else:
         sourcefile = __file__
-    with open(sourcefile, 'r', encoding='utf-8') as fp:
+    with open(sourcefile, "r", encoding="utf-8") as fp:
         test_source = fp.read()
-    example_tests = test_source[test_source.find('## BEGIN EXAMPLE TESTS'):test_source.find('## END EXAMPLE TESTS')]
+    example_tests = test_source[
+        test_source.find("## BEGIN EXAMPLE TESTS") : test_source.find(
+            "## END EXAMPLE TESTS"
+        )
+    ]
 
     missing = []
-    for line in to_test.strip().split('\n'):
+    for line in to_test.strip().split("\n"):
         if ("exampletest_single('%s')" % line) not in example_tests:
```

Comment:
Tests are failing because you changed all the testcases to use double quotes but didn't change this line which asserts testcases are present in this file by scanning for specific strings containing single quotes.